install/kubernetes/helm/istio/values.yaml

```diff
@@ -170,7 +139,7 @@ global:
 
     # Controls number of Proxy worker threads.
     # If set to 0 (default), then start worker thread for each CPU thread/core.
-    concurrency: 0
+    concurrency: 2
```

Comment:
2 is still high. you get one main thread plus 2 worker threads, on a flimsy sidecar process. envoy will eat both cores.. isn't it better to keep this to 1?

Reply:
Is it possible to do 1 for demo, 2 for prod?